producer/start-daemon-deploy.sh

```diff
@@ -1,13 +1,9 @@
 #!/bin/bash
-. /home/saluser/.setup_sal_env.sh
+. /home/saluser/.setup_dev.sh
```

Comment:
Is this line ok even do this is the "deploy" script to run things? from Inria we don't use this one